### ISSUE TITLE
Plugin incompatible with current IntelliJ

### ISSUE THREAD

**Issue by Jeetah** [wrike/sprinter-idea-plugin#6]
Just discovered and loved this great plugin - but unfortunately it is already incompatible with current IntelliJ release. Are the plans to make it compatible?

**Comment by jorgematias-teya** [comment]
Yeah, it's not compatible with 2024.2.x - I wanted to give the plugin a try but it's not compatible with my intellij version. Any plans on this? Thanks!

<img width="1095" alt="Screenshot 2024-12-17 at 10 40 32" src="https://github.com/user-attachments/assets/2a42a5db-1bcf-4dec-b896-6222ec48d10a" />
